docs/Tutorial_DiscordMERLIN.ipynb

```diff
@@ -0,0 +1,1911 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.        cands = np.flatnonzero(is_cands)
</pre><p>In <code>_prune_candidates</code>  this is <code>cands_idx = np.flatnonzero(is_cands)</code>  so please decide which is more accurate from a naming perspective. They both refer to the same thing, right?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
Thanks for catching that! I will also change the iterator `idx` to `i` (to be more consistent with previous functions)

Reply:
<p>I'd actually prefer <code>cands_idx</code>  and <code>for idx in cands_idx</code> . This is because <code>cands_idx</code>  can be non-continuous</p>

Reply:
Got it. I will do `for idx in cands_idx`.